### DISCUSSION TITLE
Ability to turn of the Brightness gesture control leaving the Volume gesture on

### DISCUSSION THREAD

**Discussion by Rockbard** [libre-tube/LibreTube#5537] [Ideas]
### Describe your suggested feature

Split the option to turn ON/OFF the Brightness and Volume in two. 
I would love the Volume gesture to work for both gesture control areas and never use the Brightness control feature. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I personally don't think we should do this, we already have way too many options in the settings and this one would be yet another niche feature.
